tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -2771,7 +2771,9 @@ def call(self, inputs, state):
     c = f * state + (1.0 - f) * x_bar
     h = r * self._activation(c) + (1.0 - r) * inputs
 
-    return h, c
+    new_state = (rnn_cell_impl.LSTMStateTuple(c, h))
```

Comment:
why this change?

Reply:
Because many other cells , such as BasicLSTMCell 、 LSTMCell 、NASCell 、 LayerNormBasicLSTMCell , return h and stateTuple with c and h when call . I think sru should return the same type as lstm .

Reply:
So I think that should be split into another PR to keep things clean.

Reply:
OK，I will do that.